### PR TITLE
Updates the Ranger Corvette to Mapping Standards

### PR DESCRIPTION
**Pull request by lavillastrangiato**
* Makes the shuttle no longer a box shuttle (sort of). Gives it power and atmos.
* Adds a nuclear reactor to the ranger corvette, and adds APCs.
* Shuffles some flags around (Xanu gets a flag!) and puts Assunzionii miscellany in the ready room.
* Updates the airlocks to have @DreamySkrell's docking 
* This PR should be considered a test of a new shuttle template. Please bug-report judiciously!
* Also a special thanks to Dreamy for helping me sort out the airlocks. This would not have been possible without them! 

image
image

(No idea why a bunch of other changelogs got caught in the changelog folder that I had to delete, BTW. Assuming it is GitKraken fuckery.)
(sorry about the commits i'm not very good at git)

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!review

**Comment by Jasorn** [comment]
does this fix the shower glass door access? if not, might as well throw that in too

**Comment by lavillastrangiato** [comment]
> does this fix the shower glass door access? if not, might as well throw that in too

Yes, it does, this is mentioned in the changelog.

**Review by DreamySkrell — APPROVED** [review]

File: html/changelogs/Lavillastrangiato - shuttlerefactor.yml
Comment (SleepyGemmy) [inline html/changelogs/Lavillastrangiato - shuttlerefactor.yml:1]: double check that this isn't a renamed changelog.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: html/changelogs/Lavillastrangiato - shuttlerefactor.yml
Comment (lavillastrangiato) [reply html/changelogs/Lavillastrangiato - shuttlerefactor.yml:1]: it's not

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**